changes/2148.feature.rst

```diff
@@ -0,0 +1 @@
+Added the `-a / --app` option to `briefcase build` and `briefcase package` to allow building/packaging a specific app instead of all apps.
```

Comment:
The changenote should be phrased in the way that you'd see a release note describing the "cool features in the new release!!1!11!". You're looking for a voice that's a lot more like "marketing copy" than a commit message:
```suggestion
The `build` and `package` commands can now be run on a single app in a project by using the `-a / --app` option.
```